src/app/routes/enrolment/view-requests/components/expiration-date/expiration-date.component.html

```diff
@@ -1,26 +1,40 @@
 <mat-label class="ml-3 pl-1">Expiration Date</mat-label>
 <app-expiration-info
   [defaultValidityPeriod]="defaultValidityPeriod"
-  [expirationTime]="expirationTimeShift"></app-expiration-info>
+  [expirationTime]="expirationTimeShift"
```

Comment:
Maybe this a side note to this PR, but does the name `expirationTimeShift` make sense to you here? Seems like a "time shift" is more of a time period than a specific time, so it's like there is a type mismatched here.

Reply:
@dawidgil

Reply:
The problem with previous implementation was that I had wrong assumption -> I though expirationTimestamp is described in seconds like defaultValidityPeriod. But expirationTimestamp is a date in miliseconds. So renaming it makes sense.

Reply:
So sounds like we should rename `expirationTimeShift` to `expirationTime`? Am I understanding this right?